### PR TITLE
Fix some odd strings and remove duplicate values

### PR DESCRIPTION
**Pull request by dkanada**
```
ButtonAdd
LabelAll
TabArtists
ButtonRepeat
ButtonHelp
ButtonSave
ButtonToggleContextMenu
ButtonSearch
TabDisplay
HeaderDisplay
HeaderFilters
HeaderFeatures
HeaderTags
TabMetadata
TabTranscoding
TabPlayback
TabSuggestions
```

### CODE REVIEW

File: src/scripts/libraryMenu.js [hunk 1]
```diff
@@ -387,7 +387,7 @@ import 'flexStyles';
             pageIds: ['mediaLibraryPage', 'librarySettingsPage', 'libraryDisplayPage', 'metadataImagesConfigurationPage', 'metadataNfoPage'],
             icon: 'folder'
         }, {
-            name: globalize.translate('TabPlayback'),
+            name: globalize.translate('TitlePlayback'),
```
Comment (MrTimscampi) [inline src/scripts/libraryMenu.js:390]: Why not rename it simply "Playback"?

Comment (dkanada) [reply src/scripts/libraryMenu.js:390]: There wasn't a duplicate string called `Playback`, only these two.

Comment (MrTimscampi) [reply src/scripts/libraryMenu.js:390]: But shouldn't we rename it to `Playback` at some point, just to make it clearer? (It's not blocking though, to be clear :p )

Comment (dkanada) [reply src/scripts/libraryMenu.js:390]: I definitely agree with that, I just didn't want to affect too much with this pull request.

**Pull request merged**